### ISSUE TITLE
continous traits whose values are strings causes auspice to crash

### ISSUE THREAD

**Issue by jameshadfield** [nextstrain/auspice#790]
Given an input metadata tsv file to `augur export` all values are exported as strings, even those which are destined to be continuous traits in auspice. This causes an app crash in auspice. 

I think this wasn't uncovered from us as all continuous values in other builds were taken from node-data JSONs rather than the metadata TSV.

**Comment by emmahodcroft** [comment]
I'm at BC2 this week so not looking into augur/auspice much ATM (will try soon), but I export age data in the EV-D68 builds from metadata as continuous (have used `export`, `export v1`, and `export v2`, though some more recently than others) without issue. So might be slightly more nuanced - just as an info-note! :)

**Comment by jameshadfield** [comment]
Yes - we ended up discovering this too - but when `augur export` decides to export them as strings, then auspice crashes.

**Comment by emmahodcroft** [comment]
James do you have any more info on when it seems augur export decides to export as strings? (And maybe example files?) Is this just an `auspice` `v5` problem, or does this also happen in `auspice `v6` `export v1`?
(`v5` issues should be focused on an `auspice` fix for existing files but also on `augur export` to stop this, it seems? And I suppose any `v6` issues should be fixed in `export` there.)

**Comment by jameshadfield** [comment]
I don't have any more information on this sorry. `augur export` (v5, at least) doesn't seem to do this very often so it's not high priority. The next time we have a dataset which triggers this we should revisit.
